### ISSUE TITLE
SequenceWindow, Ile over-representation on the n-term side

### ISSUE THREAD

**Issue by pisistrato** [Nesvilab/FragPipe#1679]
Hi,

I was inspecting the files generated in the tmt-report folder. I noticed a suspicious over-representation of Ile in the SequenceWindow on the n-term side, i.e. before the detected peptide sequence.
Can you comment on how that is `calculated`?
It might be real, but I was expecting a Leu to be over-represented...

**Comment by fcyu** [comment]
It just uses the sequence of the assigned protein in the fasta file.

Best,

Fengchao

**Comment by pisistrato** [comment]
Since it was very strange, I checked the sequences manually, this is what I see
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/ga87zix/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/ga87zix/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#467886" vlink="#96607D">


ProteinID | SequenceWindow | Start | SequenceWindowFromFasta | Fasta
-- | -- | -- | -- | --
A0A8I5KX85 | TAPVQAPPAP | 148 | TAPVQAPPAP | xxxxxMAETEERSLDNFFAKRDKKKKKERSNRAASAAGAAGSAGGSSGAAGAAGGGAGAGTRPGDGGTASAGAAGPGAATKAVTKDEDEWKELEQKEVDYSGLRVQAMQISEKEEDDNEKRQDPGDNWEEGGGGGGGMEKSSGPWNKTAPVQAPPAPVIVTETPEPAMTSGVYRPPGARLTTTRKTPQGPPEIYSDTQFPSLQSTAKHVESRKDKEMEKSFEVVRHKNRGRDEVSKNQALKLQLDNQYAVLENQKSSHSQYNxxxxx
A0A8I5KX85 | AIKIQLDNQY | 239 | ALKLQLDNQY | xxxxxMAETEERSLDNFFAKRDKKKKKERSNRAASAAGAAGSAGGSSGAAGAAGGGAGAGTRPGDGGTASAGAAGPGAATKAVTKDEDEWKELEQKEVDYSGLRVQAMQISEKEEDDNEKRQDPGDNWEEGGGGGGGMEKSSGPWNKTAPVQAPPAPVIVTETPEPAMTSGVYRPPGARLTTTRKTPQGPPEIYSDTQFPSLQSTAKHVESRKDKEMEKSFEVVRHKNRGRDEVSKNQALKLQLDNQYAVLENQKSSHSQYNxxxxx
A0A8I5KX85 | NQAIKLQLDN | 237 | NQALKLQLDN | xxxxxMAETEERSLDNFFAKRDKKKKKERSNRAASAAGAAGSAGGSSGAAGAAGGGAGAGTRPGDGGTASAGAAGPGAATKAVTKDEDEWKELEQKEVDYSGLRVQAMQISEKEEDDNEKRQDPGDNWEEGGGGGGGMEKSSGPWNKTAPVQAPPAPVIVTETPEPAMTSGVYRPPGARLTTTRKTPQGPPEIYSDTQFPSLQSTAKHVESRKDKEMEKSFEVVRHKNRGRDEVSKNQALKLQLDNQYAVLENQKSSHSQYNxxxxx
A0A8I5KX85 | FPSIQSTAKH | 199 | FPSLQSTAKH | xxxxxMAETEERSLDNFFAKRDKKKKKERSNRAASAAGAAGSAGGSSGAAGAAGGGAGAGTRPGDGGTASAGAAGPGAATKAVTKDEDEWKELEQKEVDYSGLRVQAMQISEKEEDDNEKRQDPGDNWEEGGGGGGGMEKSSGPWNKTAPVQAPPAPVIVTETPEPAMTSGVYRPPGARLTTTRKTPQGPPEIYSDTQFPSLQSTAKHVESRKDKEMEKSFEVVRHKNRGRDEVSKNQALKLQLDNQYAVLENQKSSHSQYNxxxxx



</body>

</html>

The first one is correct, the others are not.
FYI, Start refers to the starting positon excluding the `xxxxx`

**Comment by fcyu** [comment]
The second one is also correct because there is a peptide `ALKLQLDNQY` in the protein. We don't distinguish `I` and `L` when mapping peptides to proteins because they have the identical mass.

Best,

Fengchao

**Comment by pisistrato** [comment]
Indeed, `ALKLQLDNQY` would be right, but FragPipe reports `AIKIQLDNQY`. To
me it seems that all `L` are converted to `I` in the fasta file  used to
create the SequenceWindow.

Case closed :)
